### PR TITLE
feature: TR-199 custom tabstops order for bottom toolbar

### PR DESCRIPTION
**Pull request by lecosson**
Related to https://oat-sa.atlassian.net/browse/TR-199

- add custom keyboard navigation to bottom bar

Required: https://github.com/oat-sa/extension-tao-nfer/pull/245

How to check:
 - create delivery with more than one test item and REBL theme enabled
 - check the focusable elements order is according to task

### CODE REVIEW

**Review by jsconan — APPROVED** [review]
Looks good.
Don't forget to create a PR on taoQtiTest to update the lib

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Review by jsconan — COMMENTED** [review]
Be careful, the version already exists on develop.
It needs to be increased when merging.

**Comment by jsconan** [comment]
Please avoid mixing version bump with your features. It will make it harder to backport it.

**Review by btamas — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

File: src/plugins/content/accessibility/keyNavigation/strategies/toolbarNavigation.js [hunk 1]
```diff
@@ -46,7 +46,11 @@ export default {
     init() {
         const config = this.getConfig();
         const $navigationBar = this.getTestRunner().getAreaBroker().getContainer().find('.bottom-action-bar');
-        const $toolbarElements = $navigationBar.find('.action:not(.btn-group):visible, .action.btn-group .li-inner:visible');
+        let $toolbarElements = $navigationBar.find('.action:not(.btn-group):visible, .action.btn-group .li-inner:visible');
+
+        if (config.reverseBottomToolbar) {
```
Comment (btamas) [inline src/plugins/content/accessibility/keyNavigation/strategies/toolbarNavigation.js:51]: Maybe it worth a test.

**Pull request merged**